src/mapper/pkg/resolvers/schema.resolvers.go

```diff
@@ -286,14 +286,7 @@ func (r *queryResolver) ServiceIntents(ctx context.Context, namespaces []string,
 	return intentsBySource, nil
 }
 
-func (r *queryResolver) Intents(
-	ctx context.Context,
-	namespaces []string,
-	includeLabels []string,
-	excludeServiceWithLabels []string,
-	includeAllLabels *bool,
-	serverName *string,
-) ([]model.Intent, error) {
+func (r *queryResolver) Intents(ctx context.Context, namespaces []string, includeLabels []string, excludeServiceWithLabels []string, includeAllLabels *bool, serverName *string) ([]model.Intent, error) {
```

Comment:
Can you make serverName into serverName + namespace (object) with both fields required?